webapp/src/components/viewTitle.scss

```diff
@@ -38,4 +38,8 @@
     > .description > * {
         flex-grow: 1;
     }
+
+    .non-breaking-space {
```

Comment:
This is not what I was expecting, I was thinking more in something like changing the already existing `.add-buttons .Icon` class and add there the `margin-right: 0.5em`, that way you don't need the `span` element at all, and you also fix the same problem with the `Hide` and `Emoji` icons.